os/board/rtl8730e/src/rtl8730e_st7789.c

```diff
@@ -34,14 +35,27 @@
 #define GPIO_PIN_BACKLIGHT 	PA_9
 #define GPIO_PIN_CMDDATA 	PA_10
 
+/****************************************************************************
+ * Private Types
+ ****************************************************************************/
+
+struct rtl8730e_st7789_info_s {
+        struct st7789_config_s lcd_config;
+        pwmout_t pwm_led;
+};
+
+
 static int rtl8730_st7789_gpio_reset(void);
 static int rtl8730_st7789_gpio_cmddata(uint8_t cmd);
 static int rtl8730_st7789_gpio_backlight(uint8_t level);
 
-struct st7789_config_s g_rtl8730_config_dev_s = {
-	.reset = rtl8730_st7789_gpio_reset,
-	.cmddata = rtl8730_st7789_gpio_cmddata,
-	.backlight = rtl8730_st7789_gpio_backlight,
+struct rtl8730e_st7789_info_s g_rtl8730_config_dev_s = {
+	.lcd_config = {
+		.reset = rtl8730_st7789_gpio_reset,
+		.cmddata = rtl8730_st7789_gpio_cmddata,
+		.backlight = rtl8730_st7789_gpio_backlight,
+	},
+	.pwm_led.pwm_idx = 1,
```

Comment:
What does it mean `1`? If possible, how about changing it to definition? or let's remove a meaning of that.

Reply:
This was removed. Is it unnecessary?

Reply:
Actually if we don't set any id, it by default take id as 0 in pwm_init function

Reply:
Yes, it was unnecessary because in pwmout_init, idx value is set already using defined structure in which it is 1.
board/rtl8730e/src/component/mbed/targets/hal/rtl8730e/pwmout_api.c 
const u32 pin2chan[20][2] = {
        {PA_7,  0},
        {PA_9,  1},
        {PA_10, 2},
        {PA_11, 3},
        {PA_12, 4},
        {PA_13, 5},
        {PA_14, 0},
        {PA_8,  2},
        {PA_4,  3},
        {PA_5,  4},
        {PA_6,  5},
        {PB_14, 0},
        {PB_15, 1},
        {PB_16, 2},
        {PB_21, 3},
        {PB_22, 4},
        {PB_25, 0},
        {PB_26, 1},
        {PB_27, 2},
        {PB_30, 3}
};